tests/unit/validators/date-test.js

```diff
@@ -217,7 +217,8 @@ test('after now or on', function(assert) {
   assert.expect(3);
   var now = moment().format('MMM Do, YYYY');
   options = {
-    onOrAfter: 'now'
+    onOrAfter: 'now',
+    percision: 'second'
```

Comment:
I noticed the milliseconds would randomly fail, when both values were now. This fixed it.